crates/tombi-schema-store/src/schema.rs

```diff
@@ -59,10 +61,25 @@ pub struct PropertySchema {
     pub property_schema: Referable<ValueSchema>,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, serde::Deserialize)]
+pub enum SchemaSpec {
+    Url(SchemaUrl),
+    Raw(SchemaUrl, Cow<'static, str>),
```

Comment:
SchemaSpec is unnecessary. Instead, use the `tombi://` schema.

When the schema is `tombi`, the built-in schema is used.

taplo has a similar [example](https://github.com/tamasfe/taplo/blob/1bb2c1881995a819d2bda7681d145709c783be1b/crates/taplo-common/src/schema/mod.rs#L30).

Reply:
Built-in schemas using the `tombi://` scheme sound convenient and are certainly worth implementing.
However, raw string-based schema registration (using something like the `SchemaSpec` mechanism) could cover a broader range of use cases.

For example, consider a scenario where a command-line tool like `cargo add` edits TOML files using tombi as its backend, keeping TOML keys neatly ordered.
In such cases, you'd need a method to use schemas quickly without relying on the internet or syscalls.
While the built-in `Cargo.toml` schema would suffice here, more minor tools or scenarios involving experimental or WIP schemas would significantly benefit from a `SchemaSpec`-like mechanism.

Reply:
Make a sence.

My suggestion is to add a method like `add_embed_source_schema(url, contents)` to SchemaStore.

This would leave most interfaces unchanged and open the way for uses like `tombi://` or `local://`.